HACKING.md

```diff
@@ -1,17 +1,30 @@
 # Hacking on snapd
 
-Hacking on snapd is fun and straightforward. The code is extensively
-unit tested and we use the [spread](https://github.com/snapcore/spread)
-integration test framework for the integration/system level tests.
+Hacking on `snapd` is fun and straightforward. The code is extensively unit
+tested and we use the [spread](https://github.com/snapcore/spread) integration test framework for the
+integration/system level tests.
 
 ## Development
 
-### Supported Go versions
+### Supported Go version
 
-From snapd 2.52, snapd supports Go 1.13 and onwards. From snapd 2.38
-to 2.52, snapd requires Go 1.9+. Versions before 2.38 support Go 1.6+.
+Go 1.13 or later is required to build `snapd`.
 
-### Setting up a GOPATH
+If you need to build older versions of snapd, please have a look at the file `debian/control` to find out what dependencies were needed at the time (including which version of the go compiler).
```

Comment:
If we want to make the go/Go naming consistent, there's one more here:

```suggestion
If you need to build older versions of snapd, please have a look at the file `debian/control` to find out what dependencies were needed at the time (including which version of the Go compiler).
```

There's also another one in line 50 (it wasn't changed by your PR so I can't mark it).

Reply:
Fixed.